qoqo/Half_Adder_Example.ipynb

```diff
@@ -0,0 +1,324 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Example: Half Adder\n",
+    "In this example we write a quantum algorithm to perform an add operation between two qubits and store the result in two qubits that act as output register.\n",
+    "\n",
+    "The truth table of the circuit is\n",
+    "| in q1 | in q2 | out q4 | out q3 | out  (integer) |\n",
```

Comment:
You need to enter an additional newline here or the truth-table will not be rendered properly